### PR TITLE
Completed the html-forms exercise. -Lindsey Laveirge

### PR DESCRIPTION
**Pull request by llaveirge**
html-forms-vid

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      <div>
+        <label for="email">Email</label>
+      </div>
+      <input type="text" id="email" name="email-address" />
+      <div>
+        <label for="why-coding">Why is coding cool?</label>
+      </div>
+      <select name="why-coding-is-cool">
+        <option id="select-option" value="select" selected="selected">
+          <label for="select-option">Select an option</label>
```
Comment (BrettAlbright) [inline html-forms/index.html]: you do not need labels for the individual options in a select, just the select itself

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
Very nice initial submission! Just a few labels in the select to remove. Well done!

File: html-forms/index.html (hunk 1 above)
Comment (llaveirge) [reply html-forms/index.html]: Oh! Will fix! 
Is that true for the radio buttons as well? The graphic on pg163 of the Duckett book may have led me astray. 
image

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**